src/pretalx/cfp/templates/cfp/event/user_submission_edit.html

```diff
@@ -8,6 +8,26 @@
         {% trans "Current state of your submission:" %}
         {% include "cfp/event/fragment_state.html" with state=submission.state %}
     </p>
+    <h5>
+        <form class="add-speaker" action="{{ submission.urls.invite }}">
+            <div class="form-group form-inline">
+                {% blocktrans trimmed count count=submission.speakers.count %}
+                    Speaker:
+                    {% plural %}
+                    Speakers:
+                {% endblocktrans %}
+                {{ submission.display_speaker_names }},
```

Comment:
As a speaker, can I somehow see speakers, who have been invited? Can I access the link again in case they lost it? Can I resend the mail? As a solution, also list all invited speakers here (gray, with tooltip); clicking an invited speaker shows the new-invitation-mail page again.

Reply:
Nope, you can't, since there is only one shared invitation token per submission. Don't think that's too important either.

Reply:
Oh, I see ;)